packages/stack/pipeline/src/index.js

```diff
@@ -24,7 +24,7 @@ class Pipeline {
   generateAll(cb) {
     async.waterfall([
       (next) => {
-        this.plugins.runActionsForEvent("pipeline:generateAll:before", {}, (err) => {
+        this.plugins.runActionsForEvent("pipeline:generateAll:before", (err) => {
```

Comment:
Comment from @iurimatias on Discord:

> what is the reason for removing the {}

Reply:
This is actually a left over from debugging, but it turns out removing this isn't an issue. In fact if `{}` is given as second parameter, it'll be converted to the callback.